templates/settings.html

```
@@ -175,254 +174,163 @@ <h6 class="alert-heading">
-                            <tbody id="remote-transfers-list">
-                                <tr>
-                                    <td colspan="5" class="text-center text-muted py-3">No active transfers.</td>
-                                </tr>
-                            </tbody>
-                        </table>
-                    </div>
+            <div id="p2p-generated-psk-panel" class="alert alert-warning mt-3 d-none" role="alert">
+                <h6 class="alert-heading mb-2">Save this PSK now</h6>
+                <p class="mb-2 small">Share this PSK only with trusted File Fridge instances that should join your private network.</p>
+                <div class="input-group">
+                    <input type="text" class="form-control font-monospace" id="p2p-generated-psk-value" readonly>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)